raddb/mods-available/couchbase

```diff
@@ -238,6 +238,23 @@ couchbase {
 		#
 		connect_timeout = 3.0
 
+		#
+		#  held_trigger_min:: If a connection is held for less than the specified
+		#  period, fire a trigger.
```

Comment:
What does "held" mean in this context.  Can all three config items be updated with less ambiguous terms.